### DISCUSSION TITLE
The vvdecapp binary is not installed by default

### DISCUSSION THREAD

**Discussion by Diyari90** [fraunhoferhhi/vvdec#144] [Q&A] [answered]
Hi, i try obtain vvdec i get this warning  
CMake Warning at cmake_install.cmake:64 (message):
  The vvdecapp binary is not installed by default any more.  To also install
  vvdecapp set -DVVDEC_INSTALL_VVDECAPP=true
and i dont get this line how to fixing it
Attention: The sample application vvdecapp is not installed by default anymore. To install the application use -DVVDEC_INSTALL_VVDECAPP=ON in cmake (install-vvdecapp=1 for make)

i will be to great-full to answer me, i search much i don't knew how to fix it Iam new to this (^_^)
![Screenshot from 2023-06-23 03-08-29](https://github.com/fraunhoferhhi/vvdec/assets/33030503/44381b86-6b07-447c-b1dc-5ce1a86e741b)

**Comment by adamjw24** [comment]
How are you building VVdeC? Please specify all the commands beginning with the checkout.

Another point is. Why would you want to install it? It is still available in the `bin/` subdirectory, just not installed.

**Reply by Diyari90** [reply]
I want use vvc in ffmpeg for streaming , yes vvdecapp it been created inside /bin,
But in root it wont install like vveceapp
In instruction tell me this Attention: The sample application vvdecapp is not installed by default anymore. To install the application use -DVVDEC_INSTALL_VVDECAPP=ON in cmake (install-vvdecapp=1 for make) but i dont know how doing it manually

**Comment by adamjw24** [comment]
> How are you building VVdeC? Please specify all the commands beginning with the checkout.

...

**Reply by Diyari90** [reply]
make release && make debug && make debug-shared && make install-release && sudo make install-release install-prefix=/usr/local && sudo make install install-prefix=/usr/local

![Screenshot from 2023-06-23 13-32-22](https://github.com/fraunhoferhhi/vvdec/assets/33030503/d2192712-bacf-4884-ab5c-a606539e4481)
![Screenshot from 2023-06-23 13-32-42](https://github.com/fraunhoferhhi/vvdec/assets/33030503/196bfaf5-6245-4567-8c60-411321a20fa9)

**Reply by adamjw24** [reply]
> make release && make debug && make debug-shared && make install-release && sudo make install-release install-prefix=/usr/local && sudo make install install-prefix=/usr/local

1st. do a `make distclean`

Than repeat and add `install-vvdecapp=1` to all your `make` calls (e.g. `make release install-vvdecapp=1`

**Reply by Diyari90** [reply]
i think its like vvencapp it should be installed in my local/bin, but it say install manually ,if you could help me with that how i install manually

**Reply by adamjw24** [reply]
And you have to specify `install-prefix=/usr/local` with the first make call. Sorry, its coutner-intuitive

**Reply by Diyari90** [reply]
ok i will and i will tell you what happen thanks

**Reply by Diyari90** [reply]
make distclean

make release install-vvdecapp=1 && make debug install-vvdecapp=1 && make debug-shared install-vvdecapp=1 && make install-release install-vvdecapp=1 && sudo make install-release install-prefix=/usr/local install-vvdecapp=1 && sudo make install install-prefix=/usr/local install-vvdecapp=1

ok, just tell me did i get your point right i past this in teminal it goes right?

**Reply by adamjw24** [reply, marked as answer]
`make install-release install-prefix=/usr/local install-vvdecapp=1` should do

**Reply by adamjw24** [reply]
And add `sudo` if you don't have the write right to `/usr/local`

**Reply by Diyari90** [reply]
![Screenshot from 2023-06-23 13-46-19](https://github.com/fraunhoferhhi/vvdec/assets/33030503/5bfc7750-9d22-407f-a9e4-9b628e538bfb)

**Comment by Diyari90** [comment]
a lot of thanks to you it been done
